packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -378,19 +377,22 @@ export default class MultiSelect extends React.Component {
                           <ListBox.MenuItem
                             key={itemProps.id}
                             isActive={isChecked}
+                            role="option"
+                            aria-selected={isChecked}
+                            tabIndex={-1}
                             isHighlighted={highlightedIndex === index}
                             title={itemText}
+                            aria-label={itemText}
```

Comment:
What is the `aria-label` needed for here? 👀

Reply:
Removed